clayui.com/content/docs/components/css-alerts.md

```diff
@@ -2,6 +2,7 @@
 title: 'Alerts'
 description: 'Alerts are used to capture the attention of the user in an intrusive way. They can be used just to say that something went right, or perhaps to say that something needs to be reviewed.'
 lexiconDefinition: 'https://liferay.design/lexicon/core-components/alerts/'
+mainTabLink: 'docs/components/alert.html'
```

Comment:
Shouldn't this be `mainTabURL`? My thinking being this isnt a Link but a URL that will be consumed by a Link

Reply:
Yeah, actually I haven't been using it with the intention of being a URL but a hyperlink, just a space to create links between documents, not necessarily being a URL, although I am practically using it for this purpose. But if you take it to the literal of the URL it would probably be a `mainTabPath`. I have no problem with continuing with this, I changed that name at least 2 times 😅

Reply:
Personally I think `url` might be better, primarily because `path` can be easily confused with file path and my first thought was you were pointing to a specific file.

or maybe `mainTabURLPath` 😂 😂

Reply:
haha yeah, finding an appropriate name is a kind of hard 😆!! An option not to confuse would be `mainTabPathname` but 🤷‍♂️.